vendor/BUILD.bazel

```diff
@@ -111,6 +113,7 @@ filegroup(
         "//vendor/github.com/prometheus/procfs:all-srcs",
         "//vendor/github.com/ryanuber/go-glob:all-srcs",
         "//vendor/github.com/sethgrid/pester:all-srcs",
+        "//vendor/github.com/sirupsen/logrus:all-srcs",
```

Comment:
It's a shame that infoblox library depends on logrus, as it's the first and only dependency that does :cry: not really much we can do about it though! 😄

Reply:
Yeap not much we can do :/